variants/backend-base/config/application.rb

```diff
@@ -16,5 +16,6 @@
 
     config.middleware.insert_before Rack::Sendfile, HttpBasicAuth
     config.action_dispatch.default_headers["Permissions-Policy"] = "interest-cohort=()"
+    config.action_dispatch.default_headers["X-Frame-Options"] = "deny"
```

Comment:
🐘 I'm 99% sure it doesn't matter, but I think it would be nice to match used in the spec + MDN docs

```suggestion
    config.action_dispatch.default_headers["X-Frame-Options"] = "DENY"
```

Reply:
I initially had `DENY` but changed to lower case to match what Rails does in https://guides.rubyonrails.org/security.html#http-security-headers